model-optimizer/mo/main.py

```diff
@@ -97,16 +97,25 @@ def print_argv(argv: argparse.Namespace, is_caffe: bool, is_tf: bool, is_mxnet:
 
 
 def prepare_ir(argv: argparse.Namespace):
-    is_tf, is_caffe, is_mxnet, is_kaldi, is_onnx = deduce_framework_by_namespace(argv)
-
     fem = argv.feManager
-    new_front_ends = []
-    if fem is not None:  # in future, check of 'use_legacy_frontend' in argv can be added here
-        new_front_ends = fem.get_available_front_ends()
+    available_moc_front_ends = []
+    moc_front_end = None
+    if fem:  # in future, check of 'use_legacy_frontend' in argv can be added here
```

Comment:
please add TODO about `use_legacy_frontend` option use in future

Reply:
probably now you can define this variable `use_legacy_frontend = True` and add complete code to cover both cases legacy and fe